keyboards/bm40hsrgb/bm40hsrgb.c

```diff
@@ -14,3 +14,27 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "bm40hsrgb.h"
+
+led_config_t g_led_config = {
```

Comment:
Allows users to compile firmware without the RGB support, if they should choose.

```suggestion
#if defined(RGB_MATRIX_ENABLE)
led_config_t g_led_config = {
```